intranet/settings/__init__.py

```diff
@@ -457,7 +457,9 @@ def get_month_seconds():
     'SCOPES': {
         'read': 'Read scope',
         'write': 'Write scope'
-    }
+    },
+    # OAuth refresh tokens expire in 30 days
+    'REFRESH_TOKEN_EXPIRE_SECONDS' : 2592000,
```

Comment:
I think it may be a good idea to limit the token expiry to a maximum of a couple of days, since applications should be requesting refresh tokens anyway. That would be more secure and shouldn't affect the end user.

Reply:
There is a different setting for setting the access token expiry [ACCESS_TOKEN_EXPIRE_SECONDS](https://django-oauth-toolkit.readthedocs.io/en/latest/settings.html#access-token-expire-seconds), which is by default set to 10 hrs.  REFRESH_TOKEN_EXPIRE_SECONDS is how long applications can request new access tokens without the user reauthenticating with their password. 
After this PR, the user would need to reauthenticate to Ion with their password every 30 days.  The reason I set it to 30 days is that some applications need access without user authentication for a long period of time (i.e. the unofficial Ion app).

Reply:
I'd recommend explicitly setting this to ```60*60*24*30``` in the settings file so that it's clear what exactly it represents, rather than an arbitrary number of seconds